### ISSUE TITLE
Error Feature `clickToLoad` Not Found

### ISSUE THREAD

**Issue by nlordell** [duckduckgo/content-scope-scripts#982]
I noticed this by chance, but when using the DDG browser for macOS, I see the following error (which appears to come from these content scope scripts) in the console on pretty much every website I open:

```
Unhandled Promise Rejection: Error: feature named `clickToLoad` was not found
```

Specifically, the line that is throwing the exception is this one:

https://github.com/duckduckgo/content-scope-scripts/blob/69c8a44a21ccbef55401ccf1586d221824041ca6/src/features/click-to-load.js#L1902

I don't know whether or not it is relevant, but I'm using the DDG browser installed from the App Store (and not the DMG).

![image](https://github.com/duckduckgo/content-scope-scripts/assets/4210206/08f56244-8312-4c48-8acb-3b2cc89a70b0)
<img width="897" alt="image" src="https://github.com/duckduckgo/content-scope-scripts/assets/4210206/aa36fb9d-f46c-431d-810d-ce2b172a7520">

**Comment by nlordell** [comment]
It isn't really causing problems for me, but I thought I'd report it anyway.

**Comment by ladamski** [comment]
@nlordell thank you for reporting this!  Which version of the browser is this happening on?

**Comment by nlordell** [comment]
I was on 1.92.0. Out of curiosity, I made sure to upgrade to 1.93.0 (I hadn't noticed there was an update in the App Store when posting this issue) which seems to resolve the problem.

Going to close as resolved in 1.93.0.
